google-cloud-pubsublite/src/main/java/com/google/cloud/pubsublite/ErrorCodes.java

```diff
@@ -25,9 +25,22 @@
       ImmutableSet.of(
           Code.DEADLINE_EXCEEDED, Code.ABORTED, Code.INTERNAL, Code.UNAVAILABLE, Code.UNKNOWN);
 
+  public static final ImmutableSet<Code> STREAM_RETRYABLE_CODES =
+      ImmutableSet.of(
```

Comment:
Can you construct this as ImmutableSet.builder().addAll(RETRYABLE_CODES).add(Code.RESOURCE_EXHAUSTED).build()?